docs: Styling Component (v9) now introduces shorthands (#26773)
microsoft/fluentui: apps/public-docsite-v9/src/Concepts/StylingComponents.stories.mdx

[Griffel devtools chrome extension](https://chrome.google.com/webstore/detail/griffel-devtools/bejhagjehnpgagkaaeehdpdadmffbigb) can be used to debug style overrides. It shows all griffel styles applied on the currently selected DOM element, including the styles that are overridden in `mergeClasses`.

### Limitations

> The Griffel's approach to styling comes with certain [limitations](https://griffel.js.org/react/guides/limitations). One of them is the lack of support for [CSS shorthand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties). To work around this, Griffel [provides a `shorthands`](https://griffel.js.org/react/api/shorthands) function, whose use is demonstrated in the next example.

Reviewer: ```suggestion
Griffel's approach to styling comes with certain [limitations](https://griffel.js.org/react/guides/limitations). One of which is the lack of support for [CSS shorthand properties](https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties). To work around this, Griffel [provides a collection of shorthand functions](https://griffel.js.org/react/api/shorthands) to write css shorthand. Their usage is demonstrated in the next example.
```
